dto/src/main/java/org/jboss/pnc/dto/ProductRelease.java

```diff
@@ -57,8 +57,10 @@ private ProductRelease(
             SupportLevel supportLevel,
             Instant releaseDate,
             String downloadUrl,
-            String issueTrackerUrl) {
-        super(id, version, supportLevel, releaseDate, downloadUrl, issueTrackerUrl);
+            String issueTrackerUrl,
+            String cpe,
+            String ppCode) {
```

Comment:
I don't recall now what is cpe, which means it should be renamed.
I know, what is ppCode, but it might be hard for others. 
Use full names here, please. To me these names are not common knowledge.